core/src/components/card/test/shape/card.e2e.ts

```diff
@@ -0,0 +1,45 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * Shape is only available in the Ionic theme.
```

Comment:
Can we add a tech debt ticket to update this? If we add the new shape variants to iOS/MD themes then these tests will become relevant.

Reply:
Shouldn't this just be updated when the shapes are added to ios/md along with the tests?

Reply:
Created [FW-6119](https://outsystemsrd.atlassian.net/browse/FW-6119) for tracking and added a comment referencing it here: https://github.com/ionic-team/ionic-framework/pull/29292/commits/648668b6363c7f7e7b7116c13755b0673a3f6471